README.md

```diff
@@ -0,0 +1,199 @@
+  - [⭐️ Show your support ](#️support)
+  - [🙏 Acknowledgments ](#acknowledgements)
+  - [📝 License ](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Booking Vespa 🛵 <a name="about-project"></a>
+
+This project is designed for the reservation of fancy vespas. If you are preparing for a special day or event and you want a cool Vespa for it, this website is the best place for you. If you have a luxurious Vespa and want to make some extra money from it, you can add it to our website and others will be able to reserve it!
+
+Here are some of the benefits of using our website:
+
+- Wide selection of Vespas to choose from
+
+The main features of this website are:
+
+- Users can see details of all the Vespas.
+- Users can reserve a Vespa for a special day.
+- Users can add their own Vespas and remove them if they want to.
+
+## Database Structure: <a name="erd"></a>
+
+<img src="https://github.com/MarkoKermi/rentals_project-back-end.git" width="650">
+
+## Project Documentation 📄 <a name="project-documentation"></a>
+
+- Front-end GitHub repository [link](https://github.com/Rudy-Menekam/Rentals-project-frontend.git)👈
+- Back-end API [link](https://github.com/MarkoKermi/rentals_project-back-end.git) 👈
+
+## Kanban Board <a name="kanban-board"></a>
+
+### Kanban Board [link](https://github.com/users/Rudy-Menekam/projects/3)
+
+### Initial state of the Kanban Board:
+
+![image](https://drive.google.com/file/d/1s9aL1XscWvnGP3t9KD-jaKd5wOis7dVM/view?usp=drive_link)
+
+### Final state of the Kanban Board:
+
+![image](https://drive.google.com/file/d/1H7apABiT-PzcKlLqTZFlk7d5OGW8FGCB/view?usp=drive_link)
+
+### Team members: 3 (Marko Kermichiev, Mohi Minul Islam Yash, Menekam Rudy)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+## Built With 🛠️ <a name="built-with"></a>
+
+- Programming Languages: Ruby, SQL.
+- Framework: Ruby on Rails.
+- Database: PostgreSQL.
+- Code Editor: VS Code.
+
+## Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- A web browser like Google Chrome.
+- A code editor like Visual Studio Code with Git and Ruby.
+
+> You can check if Git is installed by running the following command in the terminal.
+
+```
+$ git --version
+```
+
+> Likewise for Ruby installation.
+
+```
+$ ruby --version && irb
+```
+
+To install rails, in the terminal kindly run this command:
+
+```
+$ gem install rails
+```
+
+### Setup
+
+Clone the repository using the GitHub link provided below.
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/Rudy-Menekam/Rentals-project-frontend.git
+```
+
+### Usage
+
+Kindly modify the files as needed.
+
+### Run locally
+
+Make your way to the correct directory by running this command:
+
+```
+$ cd Program-Capstone
+```
+
+Install the required dependencies to run the project with this command:
+
+```
+$ bundle install
+```
+
+Setup the database, run:
```

Comment:
- [ ] This readme contains setup instructions that are intended for the backend project.  

Since this is the frontend repo project, kindly include instructions for setting up and running the frontend project correctly. 👍